src/ui.rs

```diff
@@ -80,7 +80,7 @@ fn use_color(choice: ColorChoice) -> bool {
     match choice {
         ColorChoice::Always => true,
         ColorChoice::Never => false,
-        ColorChoice::Auto => atty::is(Stream::Stdout),
+        ColorChoice::Auto => io::stdout().is_terminal(),
```

Comment:
Interesting, so this would let us take https://github.com/martinvonz/jj/pull/672#discussion_r1025879900?

Reply:
Yep, I believe so.